src/components/app/index.tsx

```diff
@@ -21,18 +21,18 @@ export default function App() {
     setActivePersonId(nextPerson?.id);
 
     if (nextPerson) {
-      highlightSwimLane(nextPerson.name);
+      highlightPerson(nextPerson.name);
     } else {
-      closeAllSwimLanes();
+      closePeopleFilters();
     }
   }, [persons]);
 
-  const onAddPerson = () => {
+  const onAddPerson = (name: '') => {
```

Comment:
This should be an equals sign instead of a colon, so the default value is an empty string? I think right now, it's saying the type of `name` is an empty string.

```suggestion
  const onAddPerson = (name = '') => {
```